deployment/secure_research_environment/remote/secure_research_desktop/scripts/write_sas_tokens.sh

```diff
@@ -0,0 +1,5 @@
+#! /usr/bin/env bash
+
+sed -i 's/^sasToken/sasToken "${BACKUP}"/' /opt/configuration/credentials-backup.secret
```

Comment:
Base64 is allowed to use "/" so I would suggest using "|" as your sed field marker.

Reply:
Changed in https://github.com/alan-turing-institute/data-safe-haven/pull/1739/commits/b2fbc44dd1aaa426e070d11e77a6b38230bab487